### ISSUE TITLE
result of paper

### ISSUE THREAD

**Issue by lalalaslj** [Linxyhaha/TDRO#6]
Excuse me, is the result in the paper based on the verification set?

**Comment by Linxyhaha** [comment]
> Excuse me, is the result in the paper based on the verification set?

The result in the paper is based on the testing set. We use validation set to choose the best hyper-parameters.
